designs/disruption-controls-by-action.md

```diff
@@ -0,0 +1,224 @@
+# Disruption Controls By Action 
```

Comment:
This is just my opinion but I am not a fan of the name "action" because this isn't really an action per-say? The action is disruption but the action is driven by a reason like consolidation or emptiness and so on. I wonder if calling it "intent" or something else makes more sense

Reply:
karpenter_disruption_eligible_nodes{consolidation_type="empty",method="consolidation"} 1
karpenter_disruption_eligible_nodes{consolidation_type="multi",method="consolidation"} 1
karpenter_disruption_eligible_nodes{consolidation_type="single",method="consolidation"} 1


Do you like method better?

Reply:
ya that sounds better to me